packages/ui/src/components/va-carousel/VaCarousel.stories.ts

```diff
@@ -1,12 +1,224 @@
+  })
+
+  await step('Should remain on slide 1 by hover on indicator', async () => {
+    await userEvent.hover(NavigatesOnPageClick[1])
+    await userEvent.unhover(NavigatesOnPageClick[1])
+    expect(slides[3].getAttribute('aria-current')).toEqual('true')
+  })
 }
 
-export const Default = defineComponent({
+export const Infinite = () => ({
   components: { VaCarousel },
-  template: '<VaCarousel/>',
+  data: () => ({ items: getItems() }),
+  template: `
+    [infinite]
+    <VaCarousel :items="items" infinite />
+    [default]
+    <VaCarousel :items="items" />
+  `,
+})
+
+Infinite.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnArrowClick = canvas.getAllByRole('button', { name: 'go previous slide' }) as HTMLElement[]
+
+  await step('Should head to slide 3 from slide 1', async () => {
+    await userEvent.click(NavigatesOnArrowClick[0])
+    expect(slides[2].getAttribute('aria-current')).toEqual('true')
+  })
+
+  await step('Should have left arrow disabled', async () => {
+    expect(NavigatesOnArrowClick[1]).toBeUndefined()
+  })
+}
+
+export const Stateful = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    [true] - should slide
+    <VaCarousel :items="items" />
+    [false] - should not slide
+    <VaCarousel :items="items" :stateful="false"/>
+  `,
+})
+
+Stateful.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const [statefulNavigatesOnPageClick, NavigatesOnPageClick] = canvas.getAllByRole('button', { name: 'go slide 2' }) as HTMLElement[]
+
+  await step('Should head to slide 2', async () => {
+    await userEvent.click(statefulNavigatesOnPageClick)
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+
+  await step('Should do nothing', async () => {
+    await userEvent.click(NavigatesOnPageClick)
+    expect(slides[4].getAttribute('aria-current')).toEqual('false')
+  })
+}
+
+export const Autoscroll = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    <VaCarousel :items="items" autoscroll :arrows="false" :indicators="false" />
+  `,
+})
+
+Autoscroll.play = async ({ step }) => {
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+
+  await step('Should change slides with no interaction', async () => {
+    expect(slides[0].getAttribute('aria-current')).toEqual('true')
+    await sleep(1000)
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+    await sleep(1000)
+    expect(slides[2].getAttribute('aria-current')).toEqual('true')
+  })
+}
+
+export const Fade = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    [fade]
+    <VaCarousel :items="items" effect="fade" />
```

Comment:
There is no fade transition, so fade is actually broken.

You can mark stories as [stale] similarly to what I did in badges, then create an issue (here's issue I did for badges leftovers - #3625).

This way we'll have an easier time focusing on stories instead of spreading too thin.